test/api/v3/integration/models/GET-model_paths.test.js

```diff
@@ -14,7 +14,7 @@ describe('GET /models/:model/paths', () => {
     await expect(user.get('/models/1234/paths')).to.eventually.be.rejected.and.eql({
       code: 400,
       error: 'BadRequest',
-      message: t('invalidReqParams'),
```

Comment:
`GET /models/:model/paths returns an error when model is not accessible or doesn't exists`

That test is failing because it's expecting `"This model does not exist."` but getting `"Invalid request parameters."`.